code/game/objects/items/weaponry.dm

```diff
@@ -79,6 +79,18 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 /obj/item/claymore/suicide_act(mob/user)
 	user.visible_message(span_suicide("[user] is falling on [src]! It looks like [user.p_theyre()] trying to commit suicide!"))
 	return(BRUTELOSS)
+	
+/obj/item/claymore/excalibur
```

Comment:
```suggestion
/obj/item/claymore/ruin/excalibur
```

move this below /obj/item/claymore/ruin and remove the force/block_chance (since it shares it with the normal ruin one) but keep the new name and desc. Then if someone wants to extend this in the future then they can just do stuff to excalibur